task_1_1.py

```diff
@@ -0,0 +1,22 @@
+"""
```

Comment:
it seems you broke the checkouting process. so, now you need to delete the files from previous labs